includes/class-wc-stripe-webhook-handler.php

```
@@ -1599,48 +1599,86 @@ public function process_checkout_session_completed( $notification ): void {
+				'Failed to retrieve checkout session with expand params.',
+				[
+					'url'      => $url,
+					'expanded' => $expand,
+					'error'    => is_wp_error( $raw_session ) ? $raw_session->get_error_message() : 'Unexpected response from Stripe API.',
+				]
+			);
 			return;
 		}
 
-		$payment_intent_id = $this->get_payment_intent_id_from_checkout_session( $checkout_session );
+		$session = new WC_Stripe_Agentic_Checkout_Session( $raw_session );
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '1650', column '18'). (undefined)

(MissingImport)